lib/net_buf/Kconfig

```diff
@@ -59,4 +59,9 @@ config NET_BUF_ALIGNMENT
 	  Default value of 0 means the alignment will be the size of a void pointer,
 	  any other value will force the alignment of a net buffer in bytes.
 
+config NET_BUF_LARGE_SIZE
+	bool "Network buffer large size"
+	help
+	  Enable support for large network buffers.
```

Comment:
IMO this should specify exactly what the larger size is.

Reply:
I really do not think we need a Kconfig for this. What do others think?